API/Services/BookService.cs

```diff
@@ -609,15 +613,10 @@ public ParserInfo ParseInfo(string filePath)
                             FullFilePath = filePath,
                             IsSpecial = false,
                             Series = series.Trim(),
+                            SeriesSort = series.Trim(),
                             Volumes = seriesIndex
                         };
 
-                        // Don't set titleSort if the book belongs to a group
```

Comment:
Can you help me understand the change here?

Reply:
The following code is dead code because string.IsNullOrEmpty(seriesIndex) is already known to be false as it's a condition for this block.

The rest of the code ensures that when a seriesSort sort is filled with the value of series. Note that this value can be an empty string. This final value is actually set in `ReadingItemService.cs` (which make any attempt to have a similar logic here redundant also). The code here affects the final outcome (in `ReadingItemService.cs`) so:
- When `Series` is set and non-empty, `SeriesSort` is set to the same value and that will be used to sort the series.
- When `Series` is empty, TitleSort is used instead.

Please check issue #1354 for details and examples of the issue this is trying to solve.

Long term, instead of copying Series into SeriesSort, we should instead transform or/and normalize it. There's two mechanisms I can think of that would be beneficial:
- Unicode normalization to remove accents, etc
- Language specific transform as part of localization (I noticed that's somewhere in the future plans). Here we'd drop leading articles like "the" in English. Ideally this is language dependend and only triggered when a language is set in the metadata.